content/src/main/content/jcr_root/_oak_index/.content.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:oak="http://jackrabbit.apache.org/oak/ns/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" jcr:primaryType="nt:unstructured">
+        <cqAuditEvent
```

Comment:
@davidjgonzalez @justinedelson I wonder if it'd make sense to do something like ensure oak index where the user could have the option to install the oak:index if it's not already installed.